### PR TITLE
Release notes for Desktop Edge 2.3.0.0

### PR DESCRIPTION
**Pull request by usha-mandya**
Mac and Windows release notes for Desktop Edge 2.3.0.0

### CODE REVIEW

File: docker-for-mac/edge-release-notes.md [hunk 1]
```diff
@@ -12,6 +12,27 @@ For information about Stable releases, see the [Stable release
...
+
+### Bug fixes and minor changes
+
+- IPv6 has been re-enabled in the embedded Linux kernel, so listening on IPv6 addresses works again. Fixed [docker/for-win#6206](https://github.com/docker/for-win/issues/6206) and [docker/for-mac#4415](https://github.com/docker/for-mac/issues/4415).
+- Fixed a bug where containers disappeared from the UI when Kubernetes context is invalid. Fixes [docker/for-win#6037](https://github.com/docker/for-win/issues/6037).
+- Fixed a file descriptor leak in `vpnkit-bridge`. Fixes [docker/for-win#5841](https://github.com/docker/for-win/issues/5841).
+Added a link to the Stable channel from the Docker Desktop UI.
```
Comment (djs55) [inline docker-for-mac/edge-release-notes.md:32]: Missing a `-` on this line?

File: docker-for-windows/edge-release-notes.md [hunk 2]
```diff
@@ -12,6 +12,44 @@ For information about Stable releases, see the [Stable release
...
+- Fixed a race condition when exposing ports.
+- Prevent the WSL 2 dialog from blocking other windows.
+- Added a check that virtualization is enabled in the BIOS.
+
+**Filesharing**
+
+- Fixed a bug when parent directories of shared folders are renamed, leading to spurious `File not found` errors. Fixes [docker/for-win#6200](https://github.com/docker/for-win/issues/6200).
+- Fixed bug where a letter and two digit root folder name causes Docker Compose to fail creating directories inside the volume. Fixes [docker/for-win#6248](https://github.com/docker/for-win/issues/6248).
+- Fixed a bug which prevented containers from seeing file updates on shared volumes, when under load and a container restarts. Fixes [docker/for-win#5530](https://github.com/docker/for-win/issues/5530#issuecomment-608804192)
+- Fixed a bug where host paths are incorrectly translated into VM paths. Fixes [docker/for-win#6209](https://github.com/docker/for-win/issues/6209).
```
Comment (djs55) [inline docker-for-windows/edge-release-notes.md:42]: I've got a new **Filesharing** item:
```
- Fixed bug receiving file events on long paths (> 260 characters). Fixes [docker/for-win#6337](https://github.com/docker/for-win/issues/6337).
```
The rest of the items below aren't filesharing-specific -- should we add a gap or a new subheading?

File: docker-for-mac/edge-release-notes.md (hunk 1 above)
Comment (usha-mandya) [reply docker-for-mac/edge-release-notes.md:32]: Aah, thanks. Will fix that.

File: docker-for-windows/edge-release-notes.md (hunk 2 above)
Comment (usha-mandya) [reply docker-for-windows/edge-release-notes.md:42]: Sure, we can add a new heading 'other fixes' similar to https://docs.docker.com/docker-for-windows/edge-release-notes/#docker-desktop-community-2230.

File: docker-for-mac/edge-release-notes.md (hunk 1 above)
Comment (usha-mandya) [reply docker-for-mac/edge-release-notes.md]: Fixed.

File: docker-for-windows/edge-release-notes.md (hunk 2 above)
Comment (usha-mandya) [reply docker-for-windows/edge-release-notes.md:42]: Added the new entry and grouped the rest of the items to the 'Other fixes' bucket. PTAL.

**Comment by usha-mandya** [comment]
@djs55 Could you let me know the number of issues we've fixed in Mac and Windows GitHub repositories? 
Here's an example https://docs.docker.com/docker-for-windows/edge-release-notes/#bug-fixes-and-minor-changes

File: docker-for-windows/edge-release-notes.md (hunk 2 above)
Comment (djs55) [reply docker-for-windows/edge-release-notes.md:42]: Looks good to me!

**Comment by djs55** [comment]
@usha-mandya I counted up the for-mac and for-win issues I know about and found:
- 1 in for-mac
- 10 in for-win

File: docker-for-windows/edge-release-notes.md (hunk 2 above)
Comment (usha-mandya) [reply docker-for-windows/edge-release-notes.md:42]: Thanks.

**Comment by usha-mandya** [comment]
Thanks Dave. I've added the total bug count to the release notes.

**Comment by djs55** [comment]
Final edge 2.3.0.0 URLs:
- https://download.docker.com/win/edge/44472/Docker%20Desktop%20Installer.exe
- https://download.docker.com/mac/edge/44472/Docker.dmg

File: docker-for-mac/edge-release-notes.md [hunk 3]
```diff
@@ -12,6 +12,29 @@ For information about Stable releases, see the [Stable release
...
+
+### Bug fixes and minor changes
+
+> Docker Desktop Edge 2.3.0.0 fixes one issue reported on the [docker/for-mac](https://github.com/docker/for-mac/issues) GitHub repository.
+
+- IPv6 has been re-enabled in the embedded Linux kernel, so listening on IPv6 addresses works again. Fixed [docker/for-win#6206](https://github.com/docker/for-win/issues/6206) and [docker/for-mac#4415](https://github.com/docker/for-mac/issues/4415).
+- Fixed a bug where containers disappeared from the UI when Kubernetes context is invalid. Fixes [docker/for-win#6037](https://github.com/docker/for-win/issues/6037).
+- Fixed a file descriptor leak in `vpnkit-bridge`. Fixes [docker/for-win#5841](https://github.com/docker/for-win/issues/5841).
```
Comment (stephen-turner) [inline docker-for-mac/edge-release-notes.md:33]: I just wanted to confirm, is this bug on Mac as well as Windows?

Comment (usha-mandya) [reply docker-for-mac/edge-release-notes.md:33]: Yup, looks like it affects both Mac and Windows based on https://github.com/docker/pinata/pull/13661/files

**Comment by usha-mandya** [comment]
Merging the PR so we can publish the release notes.

**Pull request merged**